### PR TITLE
 Apply example_ynh

### PR DESCRIPTION
**Pull request by yalh76**
## Problem
- *to solve CI error and more*

## Solution
- *And how you fix that*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : Maniack C
- [x] **Approval (LGTM)** : Maniack C
- [x] **Approval (LGTM)** : JimboJoe
- **CI succeeded** : [Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/my_webapp_ynh%20PR31%20(Official_fork)/)  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by maniackcrudelis** [comment]
Code review OK, however this PR does not fix the CI.

**Comment by yalh76** [comment]
I pushed another pull request before this one just adding the integration level: https://github.com/YunoHost-Apps/my_webapp_ynh/pull/30
This push request was failed:
image
Because of: https://travis-ci.org/YunoHost-Apps/my_webapp_ynh/builds/481615933?utm_source=github_status&utm_medium=notification
image

I made all modification to let the automatic check goes fine

**Comment by frju365** [comment]
I change base branch. To be honnest, I totally agree with this PR, but is it possible to start from a clean PR ? :) I'm not sure every commit belong to the PR :)

**Comment by yalh76** [comment]
I'm a bit newbie on github and Yunohost, so I tried to fix the issue for continuous integration and make the package more compliant. So do what you think the better.

File: manifest.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     },
     "version": "1.0",
     "url": "https://github.com/YunoHost-Apps/my_webapp_ynh",
-    "license": "GPLv3",
+    "license": "free",
```
Comment (alexAubin) [inline manifest.json]: Uh I don't know exactly what's the standard to put in this field, but is it really what we want to do ?

Comment (yalh76) [reply manifest.json]: It seems that licence must be: free or non-free, as descibed in the documentation
https://yunohost.org/#/packaging_apps_manifest_en

Comment (yalh76) [reply manifest.json]: But after checking the https://github.com/YunoHost/package_linter/blob/master/package_linter, other value can be accepted if they are part of https://spdx.org/licenses/
So other good value would be:

- GPL-3.0-only
- GPL-3.0-or-later

Comment (yalh76) [reply manifest.json]: yes there is no upstream app for this package but the maintainer added a GPL3 License in the package... 
So "free" may be the good value

**Comment by maniackcrudelis** [comment]
Bump

No big deal, #34 is following.

**Comment by JimboJoe** [comment]
The CI failed, though...

**Comment by maniackcrudelis** [comment]
Let see the whole picture with #34 merged as well.

Will be merged in 3 days.

**Pull request merged**